lib/tasks/users.rake

```diff
@@ -12,6 +14,49 @@ namespace :users do
   task update_user_roles_to_standard: :environment do
     update_user_roles
   end
+
+  desc "Delete user accounts with no name or organisation"
+  task delete_users_with_no_name_or_org: :environment do
+    auth0_client = Auth0Client.new(
```

Comment:
After talking with @james-sheppard-gov-uk and @AP-Hunt, we don't want to grant access to the application in auth0 to do this deletion, as we don't currently have a way of rotating the client secret - so if it was leaked we want to limit the damage that can be done with it. So this task shouldn't attempt to delete data from auth0.